src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -465,51 +465,81 @@ private void RefreshFileList(ObservableCollection<StartPageListItem> files,
             }
         }
 
-        private Dictionary<string, object> DeserializeJsonFile(string filePath)
+        private readonly string[] jsonKeys = { "Description", "Thumbnail", "Author" };
+        private readonly Dictionary<string, object> propertyLookup = [];
+        private static DefaultJsonNameTable propertyTable = null;
+
+        private class GraphData
         {
-            if (DynamoUtilities.PathHelper.isValidJson(filePath, out string jsonString, out Exception ex))
-            {
-                return JsonConvert.DeserializeObject<Dictionary<string, object>>(jsonString);
-            }
-            else
+            public string Author = "";
+            public string Description = "";
+            public string Thumbnail = "";
+        }
+
+        private GraphData DeserializeJsonFileTest(string filePath)
+        {
+            return JsonConvert.DeserializeObject<GraphData>(File.ReadAllText(filePath));
```

Comment:
so ? was this any faster/slower than streaming?

Reply:
So if these are the numbers : https://github.com/DynamoDS/Dynamo/pull/15759/files#r1932364787
Am I to understand that `JsonConvert.DeserializeObject` is faster?